rpc/common/moc_common_storageinfo.proto

```diff
@@ -11,6 +11,7 @@ import "moc_common_common.proto";
 message StorageContainerInfo {
 	Capacity capacity  = 1;
 	Latency latency	= 2;
+	string preferredOwner = 3;
```

Comment:
If this applies for MOC cloudagent interface, then it should be reflecting zone.
Also the name should not sound specific to failover cluster.

Reply:
1. In VM creation flow, preferring container which is on same node first (determined by most preferred owner). If not, preferring containers in same zone. 
2. So, we need to maintain 'most preferred node/owner' for container and in cloud agent zone information is derived from node during container selection.
3.  If requirement is to expose 'zone' name in container spec, then I can either derive it from preferred owner in cloud agent or fetch it from volume's custom property set by ECE action plan on node agent side . Please let me know the preferred approach.

Reply:
Discussed offline. I will update cloud agent logic to populate the zone name in the container info.